src/components/app.jsx

```diff
@@ -47,6 +47,12 @@ const App = React.createClass({
               </div>
             </section>
             {this.props.children}
+            <footer style={{marginTop: "5em"}}>
+              <hr/>
+              <a href={config.gitWebpageUrl} rel="external" target="_blank">
```

Comment:
I thought it was a good practice to put it when it's an external link. OK to remove.